### PR TITLE
Feature/glue connection resource

### PR DESCRIPTION
**Pull request by atxplangl**

### CODE REVIEW

File: glue_connection.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+resource "aws_glue_connection" "this" {
+  count = var.enable_glue_connection ? 1 : 0
+  name  = "${var.name}-${random_string.identifier.result}"
+  tags  = merge(local.tags, var.tags)
+  connection_properties = {
+    JDBC_CONNECTION_URL = "jdbc:mysql://${aws_rds_cluster.this.endpoint}/${aws_rds_cluster.this.database_name}"
```
Comment (reeves122) [inline glue_connection.tf:6]: I think port needs to be here too. 
`"jdbc:mysql://${aws_rds_cluster.this.endpoint}:${port}/${aws_rds_cluster.this.database_name}"`

**Review by reeves122 — CHANGES_REQUESTED** [review]

File: glue_connection.tf (hunk 1 above)
Comment (plangley-champ) [reply glue_connection.tf:6]: added port

File: glue_connection.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    PASSWORD            = aws_secretsmanager_secret_version.this[0].secret_string
```
[the remark this answers is not in the document]
Comment (plangley-champ) [reply glue_connection.tf:7]: removed secrets manager

File: variables.tf [hunk 3]
```diff
@@ -384,5 +384,24 @@ variable "vpc_id" {
...
+  default     = {}
+}
+
+variable "enable_glue_connection" {
+  description = "enable glue connection for rds"
+  type        = bool
+  default     = false
+}
+
+variable "name" {
```
[the remark this answers is not in the document]
Comment (plangley-champ) [reply variables.tf:399]: using cluster identifier instead

**Review by reeves122 — APPROVED** [review]

**Pull request merged**